x-pack/plugins/gis/public/shared/layers/sources/source.js

```diff
@@ -43,6 +43,15 @@ export class ASource {
     return '';
   }
 
+  /**
+   * return attribution for this layer as array of objects with url and label property.
+   * e.g. [{ url: 'example.com', label: 'foobar' }]
+   * @return {Promise<null>}
+   */
+  async getAttributions() {
```

Comment:
Why is this method `async`? Neither implementation uses `await`

Reply:
I wanted to future-proof this now, as we might want to resolve this dynamically in some cases (e.g. suppose we add an ajax data source, or a way to add attribution from ES, ...). This is similar to why `getDisplayName` is async (ie. because for ES-sources this is an async lookup).